crystalBox.py

```diff
@@ -236,13 +270,31 @@ def get_scatterers(self):
 
         print('getting scatterers')
         scatterers = self.scatterers
-        print(scatterers)
+        #print(scatterers)
         scatterers = [atm.split(' ') for atm in list(scatterers)]
         scatterers = [[val if val.isalpha() else float(val) \
                        for val in scatterer] for scatterer in scatterers]
         print(scatterers)
         return scatterers
     
+    def set_scatterers(self):
+        scatterers = self.estimate_uiso()
+        #return [f'{scatterer[0]} {scatterer[1]:.5f} {scatterer[2]:.5f} {scatterer[3]:.5f} {scatterer[4]:.0f} {scatterer[5]:.5f}' for scatterer in scatterers]
+        return "; ".join([f'{scatterer[0]} {scatterer[1]} {scatterer[2]} {scatterer[3]} {scatterer[4]} {scatterer[5]}' for scatterer in scatterers])      
+    
+    def estimate_uiso(self):
+        #sampleName = crys.Box('kdp')
+        CellContents=self.get_scatterers()
```

Comment:
make `CellContents` a class attribute so user can access content. Also, rename to `cellContents`